src/ring_theory/graded_algebra/homogeneous_ideal.lean

```diff
@@ -257,23 +274,29 @@ instance : has_bot (homogeneous_ideal 𝒜) :=
 @[simp] lemma coe_bot : ↑(⊥ : homogeneous_ideal 𝒜) = (⊥ : ideal A) := rfl
 
 @[simp] lemma eq_bot_iff (I : homogeneous_ideal 𝒜) : I = ⊥ ↔ (I : ideal A) = ⊥ :=
-subtype.ext_iff
+begin
+  rw show I = ⊥ ↔ I.to_ideal = ⊥, from ⟨λ hI, by rw hI; refl, λ hI, by ext1; exact hI⟩,
+  refl
+end
```

Comment:
`to_ideal.injective.eq_iff` should close this

Reply:
yes